### ISSUE TITLE
Cannot connect to AP on RTL7871

### ISSUE THREAD

**Issue by megagunner786** [wi-fi-analyzer/fluxion#13]
### Which version of fluxion are you using?
Version 2, rev 9

### What distribution of Linux (including the version) are you running it on?
Linux 2017.1 32 bit

### What wireless adapter are you using? Exact model and chipset, statements like "internal" and "was working before" are not helpful at all.
Thomson TG123G, Realtek RTL8187B Wireless 802.11b/g 54Mbps USB 2.0 Network Adapter

### Which is the driver for it?
RTL8187 driver

### Does it support injection ( output of aireplay-ng -9)
It does.
Output is
Injection is working!
### General description of your issue along with the steps to reproduce it
Basically, I'm successful in capturing a handshake of my network, after creating a fake AP, I am unable to connect to it.

1) Capture handshake
2) Create SSL certificate
3) Choose the web interface option
4) Create the fake AP - with English as the page
5) Try to connect to it on your phone

**Comment by fabras78** [comment]
I stolen the handshake but when I'm waiting for connection from stations, I use my mobile or workstation but I can't see any reconnection.... I still continue to use internet on my network and no reconnection is requested... why?
I need to do anything else?

**Comment by azamet90** [comment]
did you fix the problem?
